audio-worklet/src/noisegate-script-processor.js

```diff
@@ -196,22 +199,22 @@ class NoiseGate {
 
   /**
    * Computes the filter coefficent for the envelope filter.
-   * @param  {Number} timeConstant the time in seconds for filter to reach
+   * @param  {Number} timeConstant The time in seconds for filter to reach
    *                               1 - 1/e of its value given a transition from
    *                               0 to 1.
-   * @param  {Number} sampleRate the number of samples per second
-   * @return {Number} alpha weight governing envelope response
+   * @param  {Number} sampleRate The number of samples per second.
+   * @return {Number} Alpha weight governing envelope response.
    */
   static getAlphaFromTimeConstant_(timeConstant, sampleRate) {
     return Math.exp(-1 / (sampleRate * timeConstant));
   }
 
   /**
    * Converts number into decibel measure.
-   * @param  {Number} linearValue the amplitude of the signal
-   * @return {Number} decibelValue the dBFS of the amplitude level
+   * @param  {Number} linearAmplitude The amplitude of the signal.
```

Comment:
If db is 10*log10(x), then the input is not a linear amplutide.  It is a power (or energy) value. Adjust comment appropriately.